src/app/views/collections/Collections.jsx

```diff
@@ -26,6 +26,7 @@ const Collections = props => {
             props.dispatch(push(`${props.rootPath}/collections/${id}/preview`));
             return;
         }
+        cookies.add("collection", id, null);
```

Comment:
Should we just add this above line 23 and remove line 24? (so that we don't need to write the same line twice?